app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -6,17 +6,37 @@ class AddressSearchController < ApplicationController
       check_or_render_not_found -> { IdentityConfig.store.arcgis_search_enabled }
 
       def index
-        render json: addresses(params[:address])
+        response = addresses(params[:address])
+        render json: response[:addresses], status: response[:status]
```

Comment:
This is totally fine as is, just noodling on some ideas:

One idea, unsure if it would be clearer or not...if we renamed `addresses:` to `json:` in the return value, we could do:

```suggestion
        response = addresses(params[:address])
        render **response
```

Another alternative would be to do an array/tuple approach
```suggestion
        addresses, status = addresses(params[:address])
        render json: addresses, status: status
```